starter-code/public/styles/style.css

```diff
@@ -0,0 +1,112 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.main {
+  background-image: url("/images/spotify-background.jpeg");
+  background-size: cover;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 700px;
```

Comment:
I'd personally assign the width in vw; eg: 50vw, and then set the min-width to 400px or something